### PR TITLE
Fix Flair not appearing due to missing this._usersInFlight

### PR DESCRIPTION
**Pull request by lukebarnard1**

### CODE REVIEW

File: src/stores/FlairStore.js [hunk 1]
```diff
@@ -48,6 +48,9 @@ class FlairStore extends EventEmitter {
             //      reject: () => {}
             //  }
         };
+        this._usersInFlight = {
+            // see _usersPending
```
Comment (ara4n) [inline src/stores/FlairStore.js]: might be worth spelling out that it's literally the same schema as _usersPending - not just a "to understand this, please look at _usersPending"

**Comment by ara4n** [comment]
lgtm mod comment

File: src/stores/FlairStore.js (hunk 1 above)
Comment (lukebarnard1) [reply src/stores/FlairStore.js]: sure!

**Comment by ara4n** [comment]
lgtm still - merging to fix flair...

**Pull request merged**